code/modules/research/rdconsole.dm

```diff
@@ -52,8 +52,15 @@ won't update every console in existence) but it's more of a hassle to do. Also,
 	var/selected_category
 	var/list/datum/design/matching_designs = list() //for the search function
 	var/disk_slot_selected = 0
+	var/guncrafting = 0
 
 
+/obj/machinery/computer/rdconsole/guncrafting
+	name = "Guncrafting Console"

```

Comment:
Change the name gunlathe console

Including the type

Stop breaking my emursions